core/module/refinements_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../../spec_helper'
+
+describe "Module#refinements" do
+  ruby_version_is "3.2" do
+    it "returns list of classes redefined in module" do
+      refined_integer = nil
+      refined_string = nil
+
+      m = Module.new do
+        refine Integer do
+          refined_integer = self
+        end
+
+        refine String do
+          refined_string = self
+        end
+      end
+
+      m.refinements.should == [refined_integer, refined_string]
+    end
+  end
+end
```

Comment:
**suggestion:** I would also add a case when there are no any refinements in a module.

Also I am curious what happens when a module includes another module that has some refinements defined. Will these refinements returned when `#refinements` is called on the child module?

Reply:
It makes sense to check it in a separate case. In this case behaviour will be stated explicitly in a test title. And will not be lost during some refactoring in future.